diff --git a/out/targets/blockmap/blockmap.js b/out/targets/blockmap/blockmap.js
index bff8c16..7cb8d31 100644
--- a/out/targets/blockmap/blockmap.js
+++ b/out/targets/blockmap/blockmap.js
@@ -5,3 +5,3 @@ const fs_1 = require("fs");
 const promises_1 = require("fs/promises");
 const zlib = require("zlib");
-const blake2_js_1 = require("@noble/hashes/blake2.js");
+const blake2_js_1 = require("@noble/hashes/blake2b");

